src/main/java/org/folio/linked/data/model/entity/Resource.java

```diff
@@ -145,6 +149,10 @@ public Resource addTypes(@NonNull org.folio.ld.dictionary.ResourceTypeDictionary
     return this;
   }
 
+  public boolean isOfType(ResourceTypeDictionary... types) {
```

Comment:
I think the behavior of this method is little confusing.
Let us assume that the type of this resource is `Identity`, `LCCN`.

When `resource.isOfType(Identity, STREN)` is invoked on this object, I would normally expect that the method will return `false`. However, the method returns true. It seems little confusing to me.

May be renaming the method to any of the following will avoid confusion?

containsAnyType
matchesAnyType
hasAnyOfTypes
isAnyTypePresent

Reply:
isAnyOfType

Reply:
I see, that's really confusing now. Initially it was a single type argument and naming was clear, right? Let me replace varargs  with single argument, it should be straightforward, agree?